src/components/Filter/Filter.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks';
+import Filter from '../Filter';
+import FilterShowcase from '../storyUtils/FilterShowcase';
+import Stack from '../storyUtils/Stack';
+
+<Meta title="Design System/Filter" component={Filter} />
+
+# Filter
+
+A universal Filter component that is a clickable element that can hold many forms.
+
+# Usage
+
+```js
+import { Filter } from '@orfium/ictinus';
+
+<Filter onClick={() => {}}>Click Me</Filter>;
```

Comment:
this usage example is not quite right based on what i saw of your example. Please use a minimal example

Reply:
yes you are right i changed it